### PR TITLE
[VIRT] VirtIO RNG testing

### PR DESCRIPTION
**Pull request by ILpinto**
RNG test:
Create VM with RNG device and check that we have virio-rng device attached.
Check that we can use /dev/random and /dev/hwrng with dd command

### CODE REVIEW

**Comment by ILpinto** [comment]
@gbenhaim 
Can you take a look the tests failing in compilation  
_+ echo 'Error: Failed to compile tests'
Error: Failed to compile tests
+ exit 2_

Locally i managed to compile and run the tests.

**Comment by ILpinto** [comment]
@gbenhaim 
Can you take a look, the testing are failing on compilation
from the log: https://jenkins.ovirt.org/job/kubevirt_kubevirt-ansible_standard-check-pr/1505//artifact/check-patch.openshift_3-10.el7.x86_64/mock_logs/script/stdout_stderr.log

+ echo 'Error: Failed to compile tests'
Error: Failed to compile tests
+ exit 2

**Comment by gbenhaim** [comment]
@ILpinto this patch https://github.com/kubevirt/kubevirt-ansible/pull/422 will solve the issue.

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
@shiywang @lukas-bednar can you please review?

File: tests/virtio_rng_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	"fmt"
+	"time"
+
+
+	"github.com/google/goexpect"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
```
Comment (lukas-bednar) [inline tests/virtio_rng_test.go]: as `ktests` please ... for consistency.

File: tests/virtio_rng_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+)
+
+const (
+	DDCommand       =  "dd count=10 bs=1024 if=/dev/%s of=/tmp/%s.txt\n"
+	CheckRNGDevice  =  "cat /sys/devices/virtual/misc/hw_random/%s\n"
+	CheckFileSize   =  "ls /tmp/%s.txt | wc -l'\n"
+	outputDevice    =  "virtio_rng.0" 
+)
+var RngAvailable    string
```
Comment (lukas-bednar) [inline tests/virtio_rng_test.go]: do you need to export these names ? everything with capital letter is visible outside of module. you should use it consciously.

Comment (ILpinto) [reply tests/virtio_rng_test.go]: fixed it to private

File: tests/virtio_rng_test.go (hunk 1 above)
Comment (ILpinto) [reply tests/virtio_rng_test.go]: DONE

**Comment by ILpinto** [comment]
ci please test

**Comment by ILpinto** [comment]
ci test please

**Pull request merged**